packages/react-meteor-data/README.md

```diff
@@ -231,6 +231,59 @@ export default withTracker({
 })(Foo);
 ```
 
+#### `useFind(cursorFactory, deps)` Accellerate your lists
```

Comment:
I would switch the `useFind` and `useSubscribe` docs here, so that `useSubscribe` is covered first.

Reply:
done